csrc/trtllm_alltoall.cu

```diff
@@ -217,10 +220,113 @@ void setMaxUsableSmCount(int64_t maxSmCount) {
   flashinfer::trtllm_alltoall::setMaxUsableSmCount(static_cast<int>(maxSmCount));
 }
 
+int64_t getPrepareWorkspaceSizePerRank(int64_t epSize) {
+  int epSize32 = static_cast<int>(epSize);
+  return flashinfer::trtllm_alltoall::moe_prepare::getMoePrepareWorkspaceSize(epSize32);
+}
+
+std::tuple<at::Tensor, c10::optional<at::Tensor>, at::Tensor, at::Tensor, at::Tensor, at::Tensor,
```

Comment:
we use `std::optional` in other pytorch kernel interfaces (both should work but it's better to keep consistency).

Reply:
Thanks, let me fix